cyder/management/commands/lib/fix_maintain.py

```diff
@@ -52,13 +38,9 @@ def main():
     # This is a temporary fix for the wireless range
     # These should be resized in maintain before migration
     print 'Fixing domains...'
-    cursor.execute("SELECT * FROM domain")
+    cursor.execute("SELECT name FROM domain")
 
-    for _, domain, _, _ in cursor.fetchall():
-        if '.in-addr.arpa' in domain or domain in ('', ' ', '.'):
+    for name in cursor.fetchall():
```

Comment:
Here's the problem. `cursor.fetchall()` returns a list of tuples. It was originally a 4-tuple, but here it was changed it to be a 1-tuple. However, the tuple doesn't get unwrapped in the loop, so it screws things up down the line. Kind of surprising that it didn't throw an error.